dashboard/app/models/sections/omni_auth_section.rb

```diff
@@ -38,7 +38,8 @@ class OmniAuthSection < Section
   def self.from_omniauth(code:, type:, owner_id:, students:, section_name: I18n.t('sections.default_name', default: 'Untitled Section'))
     oauth_section = with_deleted.where(code: code).first_or_create
 
-    oauth_section.name = section_name || I18n.t('sections.default_name', default: 'Untitled Section')
+    course_name_limit = type == Section::LOGIN_TYPE_CLEVER ? CleverSection.column_for_attribute(:name).limit : GoogleClassroomSection.column_for_attribute(:name).limit
```

Comment:
Not sure what the best practice is here, but CleverSection and GoogleClassroomSection both extend OmniAuthSection, and they all extend the base Section model, and end up in the same table. Are we safe just using the limit for the OmniAuthSection field since ultimately they all have to conform to the char limit for Section?

Reply:
That sounds good to me